Update index.md (#4650)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.1/release-notes/2.1.4/index.md

| Traefik ForwardAuth | traefik-forward-auth | 0.3.2 | - chart: 0.3.2<br>- traefik-forward-auth: 3.0.2 |
| Velero | velero | 3.1.5 | - chart: 3.1.5<br>- velero: 1.5.2 |

## Known Issues

> ### Konvoy ProviderID Changes

Reviewer: ```suggestion
### Konvoy ProviderID Changes

```
I'm guessing this is left blank for a reason unbeknownst to me

Author: Just a placeholder to get the PR started. I updated it now.

> ### Konvoy ProviderID Changes

Author: ```suggestion
### Konvoy migration respects providerIDs

When attempting to migrate existing vSphere 1.8 clusters to 2.1, this ends in failure due to providerID issues with the migration. The problem occurred when migrating a 1.8 cluster with a cloud-privider who assigned providerIDs to each node. Originally we would replace these with D2iQ IDs, but now we preserve the providerIDs.
```

> Attempting to migrate existing vSphere 1.8 clusters to 2.1 results in failure. The problem occurred when migrating a 1.8 cluster with a cloud-privider who assigned providerIDs to each node. Originally we would replace these with D2iQ IDs, but now we preserve the providerIDs.

Reviewer 2: ```suggestion
Attempting to migrate existing vSphere 1.8 clusters to 2.1 previously resulted in failure. The problem occurred when migrating a 1.8 cluster with a cloud-provider who assigned providerIDs to each node. Originally we would replace these with D2iQ IDs, but now we preserve the providerIDs.
```

> Attempting to migrate existing vSphere 1.8 clusters to 2.1 results in failure. The problem occurred when migrating a 1.8 cluster with a cloud-privider who assigned providerIDs to each node. Originally we would replace these with D2iQ IDs, but now we preserve the providerIDs.

Author: ```suggestion
Previously, attempting to migrate existing 1.8 clusters to 2.1 ended in failure due to providerID issues with the migration. The problem occurred when migrating a 1.8 cluster with a cloud-provider that assigned providerIDs to each node. Originally we would replace these with D2iQ IDs, but now we preserve the providerIDs, so that these migrations now succeed.
```

> When attempting to migrate existing vSphere 1.8 clusters to 2.1, this ends in failure due to providerID issues with the migration. The problem occurred when migrating a 1.8 cluster with a cloud-privider who assigned providerIDs to each node. Originally we would replace these with D2iQ IDs, but now we preserve the providerIDs.

Author: ```suggestion
Attempting to migrate existing vSphere 1.8 clusters to 2.1 ends in failure. The problem occurred when migrating a 1.8 cluster with a cloud-privider who assigned providerIDs to each node. Originally we would replace these with D2iQ IDs, but now we preserve the providerIDs.
```

> When attempting to migrate existing vSphere 1.8 clusters to 2.1, this ends in failure due to providerID issues with the migration. The problem occurred when migrating a 1.8 cluster with a cloud-privider who assigned providerIDs to each node. Originally we would replace these with D2iQ IDs, but now we preserve the providerIDs.

Reviewer 2: ```suggestion
Attempting to migrate existing vSphere 1.8 clusters to 2.1, ended in failure due to providerID issues with the migration. The problem occurred when migrating a 1.8 cluster with the cloud-provider (such as vSphere) that assigned providerIDs to each node. Originally we would replace these with D2iQ IDs, but now we preserve the providerIDs, so that these migrations now succeed.
```

> Previously, attempting to migrate existing 1.8 clusters to 2.1 ended in failure due to providerID issues with the migration. The problem occurred when migrating a 1.8 cluster with a cloud-provider that assigned providerIDs to each node. Originally we would replace these with D2iQ IDs, but now we preserve the providerIDs, so that these migrations now succeed.

Author: ```suggestion
Previously, attempting to migrate existing 1.8 clusters to 2.1 ended in failure due to providerID issues with the migration. The problem occurred when migrating a 1.8 cluster with a cloud-provider that assigned providerIDs to each node. Originally, we would replace these with D2iQ IDs, but now we preserve the providerIDs so that these migrations succeed.
```

| Traefik ForwardAuth | traefik-forward-auth | 0.3.2 | - chart: 0.3.2<br>- traefik-forward-auth: 3.0.2 |
| Velero | velero | 3.1.5 | - chart: 3.1.5<br>- velero: 1.5.2 |

## Known Issues

### Konvoy migration preserves providerIDs

When attempting to migrate existing vSphere 1.8 clusters to 2.1, this ends in failure due to providerID issues with the migration. The problem occurred when migrating a 1.8 cluster with a cloud-privider who assigned providerIDs to each node. Originally we would replace these with D2iQ IDs, but now we preserve the providerIDs.

> 
> <content>

Author: ```suggestion
```
